src/views/Correction.vue

```diff
@@ -75,12 +75,16 @@
                 <p>Enter a detail that will appear on the ledger for this entity.</p>
                 <p class="black--text mb-0">{{defaultComment}}</p>
               </header>
-              <DetailComment
-                v-model="detailComment"
-                placeholder="Add a Detail that will appear on the ledger for this entity."
-                :maxLength="maxDetailCommentLength"
-                @valid="detailCommentValid=$event"
-              />
+              <v-card flat>
+                <div no-gutters class="px-4 py-2">
```

Comment:
Do you need `no-gutters` here? I usually see it used for v-rows, which do something weird with left/right (negative) margins.

You might also be able to put the px/py classes right in the DetailComment element below.

Reply:
Just removed `no-gutter` and it didn't change anything. I'll add a new commit removing it.

I think the px/py classes have to be outside the DetailComment to not affect it in other places, see my comment below for more details.

Reply:
I misunderstood I see what you mean by adding it right in the DetailComment. I added it and it works so I should be good to remove the div.